tests/unit/datarobot_drum/drum/test_drum.py

```diff
@@ -618,3 +618,35 @@ def test_runtime_params_not_present_as_env_var(
             assert file_mapping_param not in docker_cmd
             assert params_file_host_location not in docker_cmd
             assert params_file_docker_location not in docker_cmd
+
+
+class TestGetDockerTagNameFromDockerContextDir:
+    """Container for tests related to the method `_create_docker_tag_name`"""
```

Comment:
nit, remove unnecessary docstring and consider rename test class name:
```suggestion
class TestCreateDockerTagName:
```